x-pack/plugins/actions/server/builtin_action_types/servicenow/service.ts

```diff
@@ -43,6 +43,14 @@ export const createExternalService = (
     return `${url}/${VIEW_INCIDENT_URL}${id}`;
   };
 
+  const checkInstance = (res: AxiosResponse) => {
+    if (res.status === 200 && res.data.result == null) {
```

Comment:
Another useful check is `res.headers['Content-Type'] === 'application/json'`. Are we sure that the result will be always there if the instance is alive? If is not alive the type of the response is `html`. I am fine with both of them. Just curious :)

Reply:
yes, result will be there if instance is alive.